From c116f0ae2c36c06fae3a72124cbe7686f930fcc1 Mon Sep 17 00:00:00 2001
From: Kirill Kapranov <kirill.kapranov@compulab.co.il>
Date: Wed, 10 Apr 2019 19:01:37 +0300
Subject: [PATCH 26/43] ucm-imx8m-mini: dts: Fix eMMC mode

Fix usdhc3 (eMMC) voltage to prevent a corresponding driver to fall back
on legacy bus mode.

Signed-off-by: Kirill Kapranov <kirill.kapranov@compulab.co.il>
---
 arch/arm64/boot/dts/compulab/ucm-imx8m-mini.dts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/compulab/ucm-imx8m-mini.dts b/arch/arm64/boot/dts/compulab/ucm-imx8m-mini.dts
index 10cd529b0874..52b8834682ee 100644
--- a/arch/arm64/boot/dts/compulab/ucm-imx8m-mini.dts
+++ b/arch/arm64/boot/dts/compulab/ucm-imx8m-mini.dts
@@ -775,6 +775,8 @@
 	pinctrl-0 = <&pinctrl_usdhc3>;
 	pinctrl-1 = <&pinctrl_usdhc3_100mhz>;
 	pinctrl-2 = <&pinctrl_usdhc3_200mhz>;
+	no-1-8-v;
+	mmc-ddr-3_3v;
 	bus-width = <8>;
 	non-removable;
 	status = "okay";
-- 
2.11.0

